src/CommandIsolator.cpp

```diff
@@ -83,6 +84,11 @@ CommandIsolatorProcess::CommandIsolatorProcess(
 
 process::Future<Option<ContainerLaunchInfo>> CommandIsolatorProcess::prepare(
     const ContainerID& containerId, const ContainerConfig& containerConfig) {
+  if (m_infos.contains(containerId)) {
+    return Failure("mesos-command-module already initialized for container");
```

Comment:
do we have guarantees from mesos that prepare will not run twice?

Reply:
In memory cgroup isolator in Mesos' code, the same behaviour is expected. 
See [line 134](https://github.com/apache/mesos/blob/master/src/slave/containerizer/mesos/isolators/cgroups/subsystems/memory.cpp)

Reply:
fine for me then!